sdk-setup/src/mb2

```diff
@@ -1675,6 +1678,23 @@ switch_to_snapshot() {
     notice "Using the '$OPT_TARGET' snapshot of the build target"
 }
 
+sign_rpms() {
+    [[ -e $STATEDIR/spec ]] || fatal "No previoud build found"
+    local retcode=0
+    rpms=$(find_rpms $all_opt "$OPT_OUTPUTDIR" "$STATEDIR/spec")
+    [[ $rpms ]] || fatal "No package found"
+    local sdksign=/usr/libexec/sdk-setup/sdk-rpmsign
+
+    if [[ ! -f $sdk-sign ]]; then
+        rpmsign --addsign ${rpms}
+    else
+        $sdk-sign $@ ${rpms}
+    fi
+    retcode=$?
```

Comment:
retcode is only used here, so better to not declare earlier and `rc` is more common variable name used elsewhere.
```suggestion
    local rc=$?
```